### PR TITLE
PEPPER-415 . update pdf

### PR DESCRIPTION
**Pull request by ssettipalli**
uploaded new pdf. 
updated access.
updated pdf link .

### CODE REVIEW

File: ddp-workspace/projects/ddp-lms/src/app/pages/how-to-participate/how-to-participate.component.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { Component } from '@angular/core';
 })
 export class HowToParticipateComponent {
 
-  consent = 'https://storage.googleapis.com/cmi-study-dev-assets/lms/pdf/7.%20Adult%20Reseach%20Consent%20Form%20V5%20TRACKED%20LMS%20April%202022.docx.pdf';
+  consent = 'https://storage.googleapis.com/cmi-study-dev-assets/lms/pdf/7.%20Adult%20Reseach%20Consent%20Form%20V7%20CLEAN%20LMS%2016Dec%202022.pdf';
```
Comment (ssettipalli) [inline ddp-workspace/projects/ddp-lms/src/app/pages/how-to-participate/how-to-participate.component.ts]: followed whats in there but wondering if prod also is referring to dev-assets !!

Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-lms/src/app/pages/how-to-participate/how-to-participate.component.ts]: I think we should move this file into the github repo.

Comment (ssettipalli) [reply ddp-workspace/projects/ddp-lms/src/app/pages/how-to-participate/how-to-participate.component.ts]: Added file to git rep under pdfs

**Review by pegahtah — APPROVED** [review]
LGTM, the only thing I am a bit worried about is `dev` in the file paths, but it seems like it is in all of them

**Comment by zyme** [comment]
Looks good!

**Review by zyme — APPROVED** [review]

**Pull request merged**